atable/src/utils.ts

```diff
@@ -0,0 +1,5 @@
+export const isHtmlString = (htmlString: string) => {
+	const $temp = document.createElement('div')
```

Comment:
Document won't always be available in a Nuxt-like context, let's use DOMParser instead.

https://stackoverflow.com/a/15458968